packages/hint-doctype/README.md

```diff
@@ -0,0 +1,73 @@
+# Check that the DOCTYPE tag of the document is valid
+
+This hint checks if the HTML is using the most modern DOCTYPE.
+
+## Why is this important?
+
+In HTML, the doctype is the required "<!DOCTYPE html>" preamble
+found at the top of all documents. Its sole purpose is to prevent
+a browser from switching into so-called “quirks mode” when rendering
+a document; that is, the "<!DOCTYPE html>" doctype ensures that the
+browser makes a best-effort attempt at following the relevant specifications,
+rather than using a different rendering mode that is incompatible
+with some specifications.
+
+## What does the hint check?
+
+This hint checks if the HTML is using the most modern DOCTYPE.
```

Comment:
We should expand this a bit more. Say that we are checking that it uses the most modern and give an example, that there's nothing before the doctype of the document, and any other check that is done.